lsp/ty.lua

```diff
@@ -0,0 +1,32 @@
+---@brief
+---
+--- https://github.com/astral-sh/ty
+---
+--- `ty`, a static type checker and language server for python.
+---
+--- The language server can be installed with `uv`:
+--- ```sh
+--- uv tool install ty
+--- ```
+return {
+  cmd = { 'ty', 'server' },
+  filetypes = { 'python' },
+  root_markers = {
+    'pyproject.toml',
+    'setup.py',
+    'setup.cfg',
+    'requirements.txt',
+    'Pipfile',
+    'ty.toml',
```

Comment:
`ty.toml` should probably be top priority given that it takes precedence over `pyproject.toml` if both are present.